src/renderer/components/Button/ButtonAction.tsx

```diff
@@ -13,48 +14,62 @@ export interface ICopyButtonProps {
 }
 
 export const BackTab: React.FunctionComponent<IActionButtonProps> = ({onClick}) => {
-    return(<button 
-        className={"back-tab"} 
+    return (<button
+        className={"back-tab"}
         onClick={onClick}>
-        <img className={"icon"} src={arrow} />
+        <img className={"icon"} src={arrow}/>
     </button>);
 };
 
 export const CopyButton: React.FunctionComponent<ICopyButtonProps> = (props: ICopyButtonProps) => {
     const [clicked, setClicked] = useState(false);
+    const [timeoutId, setTimeoutId] = useState<number | null>(null);
 
     const handleClick = (): void => {
         props.onClick();
-        const handleTimeout= (): void => {
+        const handleTimeout = (): void => {
             setClicked(false);
         };
         setClicked(true);
-        setTimeout(handleTimeout,1000);
+        setTimeoutId(window.setTimeout(handleTimeout, 1000));
     };
 
-    return(<button
-        className={"copy-button"} 
+    useEffect(() => {
+        return function cleanup(): void {
+            if (timeoutId) {
+                window.clearTimeout(timeoutId);
+            }
+        };
+    });
```

Comment:
You should memoize `timeoutId`. Also, I don't think you need to return a function...